go.mod

```diff
@@ -15,10 +15,13 @@ require (
 	github.com/stretchr/testify v1.5.1
 	github.com/undefinedlabs/go-mpatch v0.0.0-20200122175732-0044123dbb98
 	github.com/vmihailenco/msgpack v4.0.4+incompatible
+	golang.org/x/lint v0.0.0-20190313153728-d0100b6bd8b3 // indirect
```

Comment:
the go.mod file doesn't need to be modified by this PR (this is due Go 1.11 that adds 3 indirections), please ensure you are using go 1.13. And you can revert this go.mod changes